funcs/cloudrun/cloudrun.go

```diff
@@ -0,0 +1,89 @@
+package cloudrun
+
+import (
+	"context"
+	"encoding/json"
+	"errors"
+	"io/ioutil"
+	"net/http"
+
+	common "github.com/athenianco/cloud-common"
+	"github.com/athenianco/cloud-common/pubsub"
+	"github.com/athenianco/cloud-common/report"
+	"github.com/athenianco/cloud-common/report/sentry"
+	"github.com/athenianco/cloud-common/service"
+)
+
+const port = ":8080"
+
+type Service interface {
+	Init() error
+}
+
+type HTTPHandler interface {
+	Service
+	http.Handler
+}
+
+type PubSubHandler interface {
+	Service
+	HandleMessage(ctx context.Context, msg *pubsub.Message) error
+}
+
+func RunHTTP(h HTTPHandler) {
+	ctx := context.Background()
+
+	defer report.Flush()
+	defer sentry.RecoverAndPanic(ctx)
+
+	if err := h.Init(); err != nil {
+		panic(err)
+	}
+	enabled, err := service.Register(ctx)
+	if err != nil {
+		panic(err)
+	}
+	if !enabled {
+		panic(report.NewIgnoredError(errors.New("disabled")))
```

Comment:
```suggestion
		report.Info(ctx, "service disabled")
		return
```
But ideally we should `return false` here and then `os.Exit(1)` in the generated code, so that CloudRun knows that service is not healthy.